src/entity/active_model.rs

```diff
@@ -45,31 +55,18 @@ where
     ActiveValue::set(v)
 }
 
-/// Defines an unset operation on an [ActiveValue]
+/// Defines an not set operation on an [ActiveValue]
 #[allow(non_snake_case)]
-pub fn Unset<V>(_: Option<bool>) -> ActiveValue<V>
+pub fn NotSet<V>() -> ActiveValue<V>
```

Comment:
I think the intended use is to `NotSet` (not a function call). Which means we should 
```rust
pub use ActiveValue::NotSet;
```

The old `Unset` function should still be kept, just being deprecated.

Reply:
That make sense

Reply:
See d0c6b5e